pkgs/top-level/all-packages.nix

```diff
@@ -2025,6 +2025,8 @@ in
 
   step-cli = callPackage ../tools/security/step-cli { };
 
+  step-ca = callPackage ../tools/security/step-ca { };
```

Comment:
(nitpick, feel free to ignore)
step-ca comes before step-cli

Reply:
Done.